### PR TITLE
#bg-163498757 Login verification bug

### PR DESCRIPTION
**Pull request by ebzeal**
#### What does this PR do?
Ensures that unverified users cannot access the portal

#### Description of Task to be completed?
The user should get notified that their account needs to be verified.

#### How should this be manually tested?
Sign up on POST /auth/signup
Do not activate email link
Try to login on POST /auth/login

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2229851/stories/163498757

### CODE REVIEW

**Review by rafmme — COMMENTED** [review]
LGTM, yet

**Comment by Bobsar0** [comment]
LGTM, yet

File: server/seeders/20190115151310-demoUser.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export default {
         email: 'kabir@now.com',
         bio: 'Learning life now',
         password: '$2y$10$QCQ1uW0OWH7xKOvJ9gNWsewzoXSjvAmXw21mcZBEB52TN6T/f2Xfy',
+        isVerified: true,
```
Comment (Sammuel09) [inline server/seeders/20190115151310-demoUser.js:28]: I am not sure why you changed this?

**Comment by Sammuel09** [comment]
LGTM yet

**Pull request merged**